CaptainJack.driver/Contents/MacOS/CaptainJack

```diff
@@ -1 +0,0 @@
-../../../build/src/captain-jack
```

Comment:
So this is necessary, though in your version it's just `../../../build/captain-jack`. Your `stage` step performs a build-in-source kind of environment which I try to stay away from.